app/src/main/java/com/asfoundation/wallet/rating/RatingInteractor.kt

```diff
@@ -9,41 +9,9 @@ import io.reactivex.Single
 import java.util.*
 
 class RatingInteractor(private val ratingRepository: RatingRepository,
```

Comment:
Should I change this in the ticket? It's not entirely related and the idea for the UseCase refactor was only for use cases directly used in the HomeViewModel, otherwise, there would be a lot more Interactor to change.